include/vrv/doc.h

```diff
@@ -120,6 +125,15 @@ class Doc : public Object {
      */
     bool GetMidiExportDone() const;
 
+    /**
+     * WG: render expansion recursion
+     */
+    xsdAnyURI_List useExpansion(xsdAnyURI_List expansionList, xsdAnyURI_List existingList, Object *prevSection);
```

Comment:
Always use `CamelCase` names (and not `camelCase` ones) for methods

Reply:
Many thanks for all your comments. I'll change that certainly.